extensions/interactions/MultipleChoiceInput/MultipleChoiceInput.js

```diff
@@ -33,7 +33,7 @@ oppia.directive('oppiaInteractiveMultipleChoiceInput', [
 
         $scope.submitAnswer = function(answer) {
           answer = parseInt(answer, 10);
-          $scope.$parent.$parent.submitAnswer(answer);
+          $scope.$parent.submitAnswer(answer);
```

Comment:
Why is this being changed?

Do other interactions (with $scope.$parent.$parent) still work fine? If not, they all should be changed.